### PR TITLE
Readme improvement-draft

### PR DESCRIPTION
**Pull request by lamiazar**
I made a draft of some `improvements` to the `readme`, I wanted to add a video tutorial and also our `ssc logo`  in the readme (but I wanted to ask first, I left some comments in this branch). there are also the `node` and `npm` versions that, I'm not sure about.

I already spoke and asked @jwerle about all that and shared a word document with him.

### CODE REVIEW

**Review by jwerle — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,25 +1,53 @@
...
+To create a new app, you may choose one of the following methods: 
+
+With **npm**:
 
-With npm:
+Any of these following commands will work
 ```bash
+npx create socket-app [react | svelte | tonic | vanilla | vue]
+npm init socket-app [react | svelte | tonic | vanilla | vue]
 npm create socket-app [react | svelte | tonic | vanilla | vue]
```
Comment (chicoxyzzy) [inline README.md:23]: I think that we need to choose only one between `npm init socket-app` or `npm create socket-app`

File: README.md [hunk 2]
```diff
@@ -1,25 +1,53 @@
...
+
 ```
 
-With yarn:
+https://user-images.githubusercontent.com/79177582/219162686-649bbdb9-015a-41bd-a613-af665a5a199e.mov  
+
+`Notes` : 
+
+`npx` is a package runner tool that comes with `npm 5.2` and above - [learn more here](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner#:~:text=npx%20is%20a%20tool%20intended%20to%20help%20round,tools%20and%20other%20executables%20hosted%20on%20the%20registry.)
```
Comment (chicoxyzzy) [inline README.md:31]: hmm, we can focus only on our tool and don't let people think too much about other tools. We should use `npm init` or `npm create` IMO

Comment (lamiazar) [reply README.md:31]: >hmm, we can focus only on our tool and don't let people think too much about other tools. We should use npm init or npm create IMO

@chicoxyzzy I'll make the change.

**Comment by lamiazar** [comment]
@chicoxyzzy I made some changes per your request, let me know what I can add or improve.

**Review by chicoxyzzy — APPROVED** [review]

**Comment by chicoxyzzy** [comment]
One more thing we can improve is adding a link to the Socket docs website

**Comment by lamiazar** [comment]
>One more thing we can improve is adding a link to the Socket docs website

I have the link to the `doc` in the `getting started` section.

File: README.md [hunk 3]
```diff
@@ -7,14 +7,13 @@ production quality app as quickly as possible.
 
 ## Getting Started 
 
-Before getting started, you may need some `Prerequisites`, to learn more you can check
-[Socket Runtime](https://sockets.sh/).
-**You'll also need to have Node>= 16 on your local developement machine.**
-That's it, you are ready to go!
+**You’ll need to have Node 16.0.0 or later version on your local development machine .**
```
Comment (chicoxyzzy) [inline README.md:10]: we also can add this to the `package.json` file so `npm` will print a warning when the version of Node is less than 16
```json
  "engines": {
    "node": ">=16"
  }
```

Comment (chicoxyzzy) [reply README.md:10]: Just put this three lines on line 8 here https://github.com/socketsupply/create-socket-app/blob/main/package.json#L8

**Comment by heapwolf** [comment]
I like this. @lamiazar I think its good to merge.

**Comment by chicoxyzzy** [comment]
I just added a minimal required version check to npm as well. Lets merge!

**Review by mribbons — APPROVED** [review]

**Pull request merged**